packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -480,6 +483,37 @@ const baseDescriptors: PropertyDescriptorMap = ArrayReduce.call(
 
 defineProperties(BaseLightningElement.prototype, baseDescriptors);
 
+// Per Component Constructor, track the corresponding Custom Element
+const ContextProviderMetaMap = new Map();
+
+function getCustomElement(Ctor: ComponentConstructor): HTMLElement {
```

Comment:
a bit unusual to see capitalized parameter name, can this be lower cased?

Reply:
sure... but we use `Ctor` everywhere in engine :)